CHANGES.rst

```diff
@@ -7,7 +7,16 @@ Changes
 `Unreleased <https://github.com/Ouranosinc/cowbird/tree/master>`_ (latest)
 ------------------------------------------------------------------------------------
 
-* Nothing yet.
+Features / Changes
+~~~~~~~~~~~~~~~~~~~~~
+
+* Basic users' workspaces management for new or removed users.
+* Add a Mongo database backend to store/restore monitoring state across sessions.
+
+Bug Fixes
+~~~~~~~~~~~~~~~~~~~~~
+* Celery has now a proper result backend.
+* Celery tasks are auto-discovered package-wide, no need to import them manually.
```

Comment:
I'm surprised about this. 
I never had to import any task manually in weaver. Everything was discovered with `config.include("pyramid_celery")`.

Reply:
The only task included in weaver seems to be here : https://github.com/crim-ca/weaver/blob/master/weaver/processes/execution.py
And this file must be included somehow before including "pyramid_celery". 

In my case the app import dynamically the services at runtime and the worker never has the chance to see that task before seeing the job on the queue. To avoid the "unregistered task" exception, celery offers https://docs.celeryproject.org/en/stable/_modules/celery/app/base.html#Celery.autodiscover_tasks, but that function required a particular package arrangement that doesn't suit cowbird.